### DISCUSSION TITLE
Party.GlobalID und Party.ID - und meine Stammaten ID.

### DISCUSSION THREAD

**Discussion by Otto404** [stephanstapel/ZUGFeRD-csharp#373] [Q&A]
Als erstes: Danke für das Projekt. 
Ansonsten: 
ich versuche gerade meine Daten auf ZugFerd abzubilden. 
Hier verwirrt mich InvoiceDescriptor.SetBuyer und im nächsten Schritt die Klasse Party.
F1: Wie unterscheiden sich ID und GlobalID?
F2: Wie/Wo kann ich die Kundennummer (die Nummer, die ich für meinen Kunden verwende) unterbringen? Ich bin es gewohnt, dass auf den Zahlungen die Rechnungsnummer und die Kundennummer angegeben wird. 

Bei InvoiceDescriptor.SetTotals - ist  totalLineItemAmount nun Brutto oder Netto?
Ist taxBasisAmount die Summe der Netto-Beträge? (mit 0, 7%, 19%)?

Warum muss ich mit AppApplicableTradeTax die Steuer noch einmal selbst eingeben; die sollte sich doch als Summe aus den LineItems ergeben. Etwa:
    var totalLineItemAmount = desc.TradeLineItems.Sum(i => i.LineTotalAmount) ?? 0;
    var tax = desc.TradeLineItems.Sum(i => i.TaxPercent * i.LineTotalAmount) ?? 0;

schon einmal vielen Dank

**Comment by stephanstapel** [comment]
Es freut mich, dass die Bibliothek nützlich ist.
Bitte habe Verständnis, dass zumindest ich inhaltliche Fragen hier nicht beantworten kann. Die Pflege der Bibliothek verbraucht viel Zeit und ich mache dies komplett in meiner Freizeit.
Solltest Du Unterstützung benötigen, biete ich das gerne in Form von Consulting an. Sollte dies für Dich interessant sein, schreib mir gerne unter vorname punkt nachname at web punkt de
